pysr/sr.py

```diff
@@ -1989,11 +2004,70 @@ def get_hof(self):
             return ret_outputs
         return ret_outputs[0]
 
+    def plot(self, ax, threshold=0.05, fontsize=10):
+        """Create a plot of the best equations.
+
+        Parameters
+         ----------
+        threshold : float
+             The plotting threshold. Plots only equations that with a score
```

Comment:
I think this metric is too restrictive. Would be better to simply plot all equations (default) or a user-passed list of indices.

Reply:
I think plotting all will be too much of a mess (in most cases). I could change it to "number of equations to plot?"

Reply:
Yeah that is a good idea! Then the default would be, say, 5 equations - the 5 with the highest score. If there are less than 5 equations, it would be equal to the number of equations. (i.e., the parameter would be default to `None` and the number inferred).